tiledb/api/c_api/attribute/test/unit_capi_attribute.cc

```diff
@@ -363,11 +364,15 @@ TEST_CASE(
   ordinary_attribute_1 attr{};
   // SECTION("success") omitted to avoid log noise
   SECTION("null context") {
-    capi_return_t rc = tiledb_attribute_dump(nullptr, attr.attribute, stderr);
```

Comment:
Should we keep minimal test coverage for the old API?

Reply:
I don't see any test for deprecated apis.